src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -3972,47 +3988,47 @@ void Fishing_UpdateFish(Actor* thisx, PlayState* play2) {
                 }
 
                 D_80B7E148 = 520.0f;
-                D_80B7E144 = 195.0f;
+                sRodLineSpooled = 195.0f;
                 SEQCMD_STOP_SEQUENCE(SEQ_PLAYER_BGM_MAIN, 10);
-                D_80B7E0A6 = 20;
-                D_80B7A6CC = 3;
+                sFishingMusicDelay = 20;
+                sFishingPlayerCinematicState = 3;
             }
             break;
 
         case 7:
-            this->unk_151 = 50;
+            this->lilyTimer = 50;
             sp134 = 5;
-            this->unk_1B0 = 12288.0f;
+            this->rotationStep = 12288.0f;
 
             if (this->actor.params < 104) {
-                this->unk_1B4 = sGroupFishes[this->actor.params - 100].pos;
-                D_80B7A898 = 1;
+                this->fishTargetPos = sGroupFishes[this->actor.params - 100].pos;
+                sFishGroupVar = 1;
             } else if (this->actor.params < 108) {
-                this->unk_1B4 = sGroupFishes[this->actor.params - 100 + 16].pos;
-                D_80B7A898 = 2;
+                this->fishTargetPos = sGroupFishes[this->actor.params - 100 + 16].pos;
+                sFishGroupVar = 2;
             } else {
-                this->unk_1B4 = sGroupFishes[this->actor.params - 100 + 32].pos;
-                D_80B7A898 = 3;
+                this->fishTargetPos = sGroupFishes[this->actor.params - 100 + 32].pos;
+                sFishGroupVar = 3;
             }
 
             Math_ApproachF(&this->actor.speedXZ, 5.0f, 1.0f, 1.0f);
 
-            if (sp124 < 20.0f) {
-                Math_ApproachS(&this->unk_170, 0x4E20, 2, 0xFA0);
+            if (distToTarget < 20.0f) {
+                Math_ApproachS(&this->fishLimbDRotZDelta, 20000, 2, 4000);
 
-                if ((this->unk_17A[2] == 0) && func_80B70A2C(this, play, false)) {
-                    func_80B71278(this, Rand_ZeroFloat(1.99f));
+                if ((this->unk_17A[2] == 0) && Fishing_SplashBySize(this, play, false)) {
+                    Fishing_FishLeapSfx(this, Rand_ZeroFloat(1.99f));
```

Comment:
Perhaps this needs a comment since `Fishing_FishLeapSfx` usually takes a boolean. I'm pretty sure since the second arg is a `u8`, I think `Rand_ZeroFloat(1.99f)` is pretty much a fifty-fifty 0 or 1 outcome (though with a slight bias to 0).